package.json

```diff
@@ -38,5 +38,7 @@
     "uglifyjs-webpack-plugin": "^1.2.7",
     "webpack": "^3.11.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "color": "^3.1.0"
```

Comment:
Is this required? Isn't a colour picker included in WordPress by default? https://github.com/WordPress/gutenberg/tree/master/packages/components/src/color-picker

Reply:
@lukecarbis this package is not the color picker, but rather used to adjust the darkness of the border in the select color display, this way the border can be a similar color variation.

Reply:
It would be really nice if we could find a workaround for this without introducing an external dependency (our only one!).

Reply:
We could use `tinycolor`, which is used in WordPress, but I did not see it exposed as usable. So it would have to be imported too.

Do you see an issue with introducing dependencies?